package.json

```diff
@@ -24,17 +26,16 @@
     "grunt-shell": "~1.1",
     "grunt-zip": "~0.16",
     "karma": "~0.12",
-    "karma-cli": "~0.0",
     "karma-chrome-launcher": "~0.1",
-    "karma-firefox-launcher": "~0.1",
```

Comment:
It wasn't. It was moved down to line 32, karma-nodeunit was moved from 35 to 33, karma-cli from 27 to 30. npm sorts the array into alphabetical order automatically. Presumably it had got out of alphabetical order due to dependencies being added to the array manually